src/services/reportCache.js

```diff
@@ -450,22 +453,40 @@ const cacheGoalMetadata = async (
     Goal.update({ onAR: true }, { where: { id: goal.id }, individualHooks: true }),
   ];
 
-  if (!isMultiRecipientReport && prompts && prompts.length) {
-    finalPromises.push(
-      cachePrompts(goal.id, arg.id, prompts),
-    );
-  }
-
   if (isMultiRecipientReport) {
```

Comment:
This code is responsible for creating the activity report goal field responses at report creation time if the fei root causes already exist on the goal.